### PR TITLE
Monthly newletter for April 2021

### PR DESCRIPTION
**Pull request by awvwgk**
I put together a draft for the April monthly newsletter and populated it from the repository activities. There has been a lot of activity at stdlib and fpm :tada:.

- [x] add stdlib updates
- [x] add fpm updates
- [x] add fortran-lang.org updates
- [x] add monthly call link
- [x] add new repository [fortran-forum-article-template](https://github.com/fortran-lang/fortran-forum-article-template)
- [x] add LLVM / classic flang updates (@AlexisPerry)
- [x] add LFortran updates (@certik)
- [x] update contributor data (@LKedward)

As usual, please add your name to the authors list if you add, edit, or review content.

### CODE REVIEW

**Comment by awvwgk** [comment]
Thanks a lot, @AlexisPerry.

File: _posts/2021-04-01-Fortran-Newsletter-April-2021.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+* [fortran-lang/fpm](https://github.com/fortran-lang/fpm)
+* [fortran-lang/fpm-registry](https://github.com/fortran-lang/fpm-registry)
+* [fortran-lang/setup-fpm](https://github.com/fortran-lang/setup-fpm)
+* [fortran-lang/fpm-haskell](https://github.com/fortran-lang/fpm-haskell)
+* [fortran-lang/fortran-lang.org](https://github.com/fortran-lang/fortran-lang.org)
+* [fortran-lang/benchmarks](https://github.com/fortran-lang/benchmarks)
+* [fortran-lang/fortran-forum-article-template](https://github.com/fortran-lang/fortran-forum-article-template)
+* [j3-fortran/fortran\_proposals](https://github.com/j3-fortran/fortran_proposals)
+
+<div id="gh-contributors" data-startdate="March 01 2021" data-enddate="April 30 2021" height="500px"></div>
```
Comment (LKedward) [inline _posts/2021-04-01-Fortran-Newsletter-April-2021.md:265]:
```suggestion
<div id="gh-contributors" data-startdate="March 01 2021" data-enddate="March 31 2021" height="500px"></div>
```

**Review by LKedward — APPROVED** [review]
Looking good! Exciting to see so much progress made in the various Fortran projects! Thanks for assembling Sebastian.

File: _posts/2021-04-01-Fortran-Newsletter-April-2021.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
+---
+layout: post
+title: "Fortran newsletter: April 2021"
+category: newsletter
+author: Sebastian Ehlert, Alexis Perry-Holby
```
Comment (awvwgk) [inline _posts/2021-04-01-Fortran-Newsletter-April-2021.md:5]:
```suggestion
author: Sebastian Ehlert, Alexis Perry-Holby, Laurence Kedward
```

**Review by milancurcic — APPROVED** [review]
I added a few events. Thank you for leading it @awvwgk!

**Comment by certik** [comment]
I'll write LFortran updates today.

**Review by certik — APPROVED** [review]
Ok, I am done. Looks good to me overall.

**Pull request merged**